.changelog/3758.txt

```diff
@@ -0,0 +1,4 @@
+```release-note:bug
+control-plane: fix an issue where ACL token cleanup did not respect a pod's GracefulShutdownPeriodSeconds and
+tokens were invalidated immediately on pod entering Terminating state.
```

Comment:
I think this might be a copy-paste error from the other PR; we want something specific here, right?

Reply:
LOOL yes thanks!